includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -811,32 +812,38 @@ public static function handle_email_change_request( $user_id ) {
 		} elseif ( \email_exists( $new_email ) ) {
 			\wc_add_notice( __( 'This email address is already in use.', 'newspack-plugin' ), 'error' );
 		} else {
-			\update_user_meta( $user_id, self::PENDING_EMAIL_CHANGE_META, $new_email );
-			// TODO: Update email with custom template.
-			\wp_mail( // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_mail_wp_mail
-				$new_email,
-				__( 'Please verify your new email address', 'newspack-plugin' ),
-				\wp_kses_post(
-					sprintf(
-						// Translators: %s is the verification link.
-						__( 'Please verify your new email address by clicking the following link: %s', 'newspack-plugin' ),
-						\add_query_arg(
-							[
-								'newspack_verify_email' => $new_email,
-								'nonce'                 => \wp_create_nonce( 'newspack_verify_email' ),
-							],
-							\home_url()
+			$update = \update_user_meta( $user_id, self::PENDING_EMAIL_CHANGE_META, $new_email );
+
+			if ( ! $update ) {
+				\wc_add_notice( __( 'Something went wrong. Please try again.', 'newspack-plugin' ), 'error' );
+			} else {
+				$sent = Emails::send_email(
+					Reader_Activation_Emails::EMAIL_TYPES['CHANGE_EMAIL'],
+					$new_email,
+					[
+						[
+							'template' => '*EMAIL_VERIFICATION_URL*',
+							'value'    => \add_query_arg(
+								[
+									self::VERIFY_EMAIL_CHANGE_PARAM => \wp_create_nonce( self::VERIFY_EMAIL_CHANGE_PARAM ),
+								],
+								\home_url()
```

Comment:
This can probably change in another PR when we handle the verification request, but we'll probably want to have this land on the Account Details page instead of home.